common/ts/sonarqube/Task.ts

```diff
@@ -35,10 +36,11 @@ export default class Task {
         if (tries <= 0) {
           throw new TimeOutReachedError();
         }
+        const errorMessage = task.errorMessage ? task.errorMessage : 'Not provided by Sonar Server';
```

Comment:
I don't see the point of having a default generic message. I would prefer to simply not append anything when there is no error message.

Reply:
Ok. I will change it today. As you have suggested

Reply:
Done